### ISSUE TITLE
addition of "Shipment/Package" to accessory category values list

### ISSUE THREAD

**Issue by bdetwiler** [bdetwiler/draft-detecting-unwanted-location-trackers#21]
// to: draft-detecting-unwanted-location-trackers@ietf.org
// from: <klein.elliot@gmail.com> 
// on: May 2, 2023

I recommend the addition of "Shipment/Package" as one additional category to list as I anticipate devices included on or within package shipments will help monitor if lost or otherwise to determine ETA or other exceptions to on-time delivery of important packages -- including vaccines, pharma products, medical devices, or other valuable shipment(s) with particular relevance into expanding B2B enterprise market use cases.

Thanks,

Elliot Klein

**Comment by gillian-epm** [comment]
Hi Elliot, is it your expectation that the shipping packages would have bluetooth tracking capability built into the package itself?  Or do you expect to be placing a BT tracker into the box?

**Comment by elliotklein** [comment]
Hi Gillian, Bluetooth monitoring for packages/shipments under the draft spec is very low cost and essentially can provide worldwide location exception monitoring for important packages. In the envisioned use case(s) the device would be placed inside a package (or potentially outside too). The value is more in **exception** monitoring, not "tracking" to know when ETA is missed by package not being in designated geofence or location(s) or other misdirected or lost package event(s) occur to then trigger a notification Ideally, by an appropriate API. 

Most value is for B2B market in monitoring shipment(s)...from high-value medical shipments, to one of a kind samples, vaccines, high value medical or biological tissue samples,, or food products that under pending U.S. and E.U. regulations will need to be location monitored for chain of custody audit and/or food safety.  

Assigning a unique category code can potentially not be only for B2B enterprise markets or other authorized shipment use cases or registered shippers (UPS, US Postal, etc.) where a unique profile is associated with the Bluetooth tag/device to better assure privacy or functionality to prevent unwanted tracking with an appropriate API or interface(s) to be developed or potentially possible in the future.

**Comment by elliotklein** [comment]
Yes, package/shipment profile considerations are good use case for pending applications for package shipments and related asset monitoring in factories or warehouse environments. Let’s  bring up again and discuss in WG or other future forums. Thanks ElliotSent from my iPhoneOn Dec 5, 2023, at 9:16 PM, Brent Ledvina ***@***.***> wrote:﻿
Right now, we're not planning on adding this to the current internet draft. We're close to forming a WG in IETF for this work -- would be good to revisit this request with the WG.
